README.md

```diff
@@ -18,97 +20,139 @@ Important notice:
+    Untar them to a folder in the home of the jira user. That folder will host your instance. I will refer that folder as `$JIRA_BINARIES`. Create an env variable if you copy-paste the commands from this guide to cmd.
 
 
-#4. Customise Jira for the Pi
+## Customise Jira for the Pi
 
  1. Set the jira home directory by manipulating some Jira files.
-      nano $JIRA_BINARIES/atlassian-jira/WEB-INF/classes/jira-application.properties
-      edit: jira.home = "/home/jira/jira_home"
-      nano $JIRA_BINARIES/bin/setenv.sh
-	  edit: JIRA_HOME="/home/jira/jira_home"
+
+	`vim $JIRA_BINARIES/atlassian-jira/WEB-INF/classes/jira-application.properties`
+
+	edit: `jira.home = "/home/jira/jira_home"`
+
+	`vim $JIRA_BINARIES/bin/setenv.sh`
+
+	edit: `JIRA_HOME="/home/jira/jira_home"`
  
  2. Set up ipv4 as preferred protocol. This will save a lot of hassle. If you know what you do, you can go with ipv6, although I ran to this issue:
-    https://community.atlassian.com/t5/Jira-questions/Tomcat-server-unavailable-on-tcp6-IPV6/qaq-p/229018
-      nano $JIRA_BINARIES/bin/setenv.sh
-	  edit: -Djava.net.preferIPv4Stack=true should be added in the JVM_REQUIRED_ARGS string
+    	https://community.atlassian.com/t5/Jira-questions/Tomcat-server-unavailable-on-tcp6-IPV6/qaq-p/229018
+
+    `vim $JIRA_BINARIES/bin/setenv.sh`
+	
+    edit: `-Djava.net.preferIPv4Stack=true` should be added in the `JVM_REQUIRED_ARGS` string
  
  3. Install the db connector for mysql
     Follow step #2 in this guide: https://confluence.atlassian.com/adminjiraserver076/connecting-jira-applications-to-mysql-945110796.html
 
 
-#5. Start Jira and attach it to your database.
+## Start Jira and attach it to your database.
 
- 1. Start jira by running $JIRA_BINARIES/bin/start-jira.sh
- 2. Connect to the web interface running on your host on the default 8080 port. You need to find out the IP of your pi first. Usurally, running the command 'hostname -I' should display your IP.
+ 1. Start jira by running `$JIRA_BINARIES/bin/start-jira.sh`
+ 2. Connect to the web interface running on your host on the default `8080` port. You need to find out the IP of your pi first. Usually, running the command 'hostname -I' should display your IP.
  3. Run the setup wizard. It may crash 2-3 times, rendering your whole Pi unresponsive. Don't give up, do a reboot and run the wizard again. It will continue. Use the wizard to connect to your mysql database.
     https://confluence.atlassian.com/adminjiraserver071/running-the-setup-wizard-802592199.html
 
 
-#6. Nice extras
+## Nice extras
+
+ 1. You might want to make sure your Jira instance starts when your Pi reboots for whatever reason. There are couple of ways to achieve this, I'd recommend using `/etc/init.d`. There's an example file named jira in this repo which you should copy to `/etc/init.d/jira`. Then run the following:
+
+    `update-rc.d jira defaults`
+ 
+    In alternative you can use systemd, create `/etc/systemd/system/jira.service`
```

Comment:
This is a warmly welcomed contribution. Please also reorder text to make systemd the nr. 1, recommended method and hint that although init.d is working, it's legacy.